### PR TITLE
Revised "Winning over executives" Docs page

### PR DESCRIPTION
**Pull request by caitlinthefirst**
## Description

Here is the "Winning Over Executives" stakeholder page to replace the current stub. This follows the format present on the other stakeholders pages. Please let me know if the messaging is off, edits needed, etc.

## Related Issues

This should be the last task on #14040.

### CODE REVIEW

File: docs/docs/winning-over-executives.md [hunk 1]
```diff
@@ -1,19 +1,58 @@
...
+Before you speak to executives about Gatsby, make sure you understand their strategic goals, and align your argument for Gatsby with those specific goals. Focus on the results of Gatsby's features rather than the specifics of how they work. If some of the big ideas behind Gatsby fit in with the executive's strategic goals, don't be afraid to include them in your conversation. 
 
-Most [benefits of Gatsby to marketers](/docs/winning-over-marketers) will also be appreciated by executives, especially revenue, conversion rates, and reduced page load times.
+Some things executives care about include:
 
-In addition, they will be interested to know that Gatsby can reduce infrastructure costs.
+- **Brand visibility** Making sure people are learning about your company by showing up in all the right places. This can be accomplished through things like SEO (search engine optimization), buying ad space, and gaining influence through content creation and/or event participation.
+- **Customer experience** Do you customers like using your site and products? Can they find what they need? Do they have a positive impression of your company?
```
Comment (marcysutton) [inline docs/docs/winning-over-executives.md]:
```suggestion
- **Customer experience** Do your customers like using your site and products? Can they find what they need? Do they have a positive impression of your company?
```

File: docs/docs/winning-over-executives.md [hunk 2]
```diff
@@ -1,19 +1,58 @@
...
 ---
 
-Executives tend to care about both top-line (revenue) and bottom-line (profit), as well as overall company brand and image.
+Before you speak to executives about Gatsby, make sure you understand their strategic goals, and align your argument for Gatsby with those specific goals. Focus on the results of Gatsby's features rather than the specifics of how they work. If some of the big ideas behind Gatsby fit in with the executive's strategic goals, don't be afraid to include them in your conversation. 
 
-Most [benefits of Gatsby to marketers](/docs/winning-over-marketers) will also be appreciated by executives, especially revenue, conversion rates, and reduced page load times.
+Some things executives care about include:
 
-In addition, they will be interested to know that Gatsby can reduce infrastructure costs.
+- **Brand visibility** Making sure people are learning about your company by showing up in all the right places. This can be accomplished through things like SEO (search engine optimization), buying ad space, and gaining influence through content creation and/or event participation.
```
Comment (marcysutton) [inline docs/docs/winning-over-executives.md]: Nice! 👍

File: docs/docs/winning-over-executives.md [hunk 3]
```diff
@@ -1,19 +1,58 @@
...
+- **Growing the business** For most executives, success means gaining clients, leads, users, purchases and awareness - getting more orders moving through that sales funnel. 
 
-You might consider sharing a story, such as [how Escalade Sports reduced their hosting bill from $5,000 to $5 using Gatsby](/blog/2018-06-14-escalade-sports-from-5000-to-5-in-hosting/), rather than giving an in-depth technical explanation of how that works.
+## Basic Explanation
 
---
+Here's an example of a basic explanation of Gatsby for executives:
 
-This is a stub. Help our community expand it.
+> Gatsby is a new way to build websites and applications with exceptional performance, reduced risk of downtime, fewer security vulnerabilities, and the flexibility to work with the best tools for your business. Companies that have have moved their sites to Gatsby have seen significant improvements in their lead conversion, organic search traffic, and team productivity.
```
Comment (marcysutton) [inline docs/docs/winning-over-executives.md]:
```suggestion
> Gatsby is a new way to build websites and applications with exceptional performance, reduced risk of downtime, fewer security vulnerabilities, and the flexibility to work with the best tools for your business. Companies that have moved their sites to Gatsby have seen significant improvements in their lead conversion, organic search traffic, and team productivity.   
```

File: docs/docs/winning-over-executives.md [hunk 4]
```diff
@@ -1,19 +1,58 @@
...
 
-Please use the [Gatsby Style Guide](/contributing/gatsby-style-guide/) to ensure your
-pull request gets accepted.
+## Specific benefits
+
+Sites built with Gatsby have experienced major sales and engagement benefits as a direct result of Gatsby's built-in features and optimizations. Here are some examples.
+
+### Better lead generation and customer engagement
+
+> Site speed an performance has a huge impact on sales and customer engagement. A [recent Akamai study](https://www.akamai.com/uk/en/about/news/press/2017-press/akamai-releases-spring-2017-state-of-online-retail-performance-report.jsp) showed that a 100 millisecond delay in a site's load time hurt conversion rates by 7%, and 53% of mobile users will leave pages that take more than 3 seconds to load. Gatsby sites are consistently 2-3x faster than similar sites built with different tools, and site owners have seen their [lead generation increase by up to 60% after transitioning to Gatsby](/blog/2018-11-16-youfit-case-study/).
```
Comment (marcysutton) [inline docs/docs/winning-over-executives.md]:
```suggestion
> Site speed and performance has a huge impact on sales and customer engagement. A [recent Akamai study](https://www.akamai.com/uk/en/about/news/press/2017-press/akamai-releases-spring-2017-state-of-online-retail-performance-report.jsp) showed that a 100 millisecond delay in a site's load time hurt conversion rates by 7%, and 53% of mobile users will leave pages that take more than 3 seconds to load. Gatsby sites are consistently 2-3x faster than similar sites built with different tools, and site owners have seen their [lead generation increase by up to 60% after transitioning to Gatsby](/blog/2018-11-16-youfit-case-study/). 
```
little typo fix!

File: docs/docs/winning-over-executives.md [hunk 5]
```diff
@@ -1,19 +1,58 @@
...
+- [Youfit uses Gatsby to increase lead generation by 60%](/blog/2018-11-16-youfit-case-study/)
+- [IBM Uses Gatsby to Manage Enterprise-Level Content](/blog/2018-12-17-ibm-case-study/#big-company-big-website)
+- [Gatsby Fuels Content and Sales in Design+Code Rebuild](/blog/2019-03-15-design-code-blog-post/)
+- [Growing Housecall Pro by 973% with Gatsby](/blog/2019-05-02-growing-housecall-pro-by-973-percent/)
+
+For even more examples of Gatsby sites, [check out the Showcase](/showcase/)
+
+## Conclusion
+
+Executives are always looking for ways to improve their business's efficiency, profits, and to grow their organization. Pointing out the financial benefits of Gatsby will help your case, but be sure to go into the many other way Gatsby will help move your business forward. If you can show your executive(s) how the benefits of Gatsby significantly outweigh the risks and costs involved in adopting it, odds are good that they'll be very receptive to the idea.
```
Comment (marcysutton) [inline docs/docs/winning-over-executives.md]:
```suggestion
Executives are always looking for ways to improve their business's efficiency, profits, and to grow their organization. Pointing out the financial benefits of Gatsby will help your case, but be sure to go into the many other ways Gatsby will help move your business forward. If you can show your executive(s) how the benefits of Gatsby significantly outweigh the risks and costs involved in adopting it, odds are good that they'll be very receptive to the idea.
```

**Comment by marcysutton** [comment]
Note: I'm making these small updates to get this merged, so no worries on fixing it!

**Comment by caitlinthefirst** [comment]
Thanks for your help @marcysutton ! Always appreciated.

**Pull request merged**